code/data_structures/src/queue/dequeue/dequeue.py

```diff
@@ -0,0 +1,32 @@
+# data_structures|queue|dequeue|python
+# Part of Cosmos by OpenGenus Foundation
+
+# Implementation using lists
+class Dequeue:
```

Comment:
I think we need `front()` and `rear` also.

Reply:
not head and tail as suggested in java's code?

Reply:
Sure, it is prefered.

Reply:
But it is about get-functions above.